.github/workflows/build-checks.yaml

```diff
@@ -95,7 +95,7 @@ jobs:
           echo "LIBSECCOMP_LINK_TYPE=static" >> $GITHUB_ENV
           echo "LIBSECCOMP_LIB_PATH=${libseccomp_install_dir}/lib" >> $GITHUB_ENV
       - name: Install protobuf-compiler
-        if: ${{ matrix.command != 'make vendor' && (matrix.component == 'agent' || matrix.component == 'runk') }}
+        if: ${{ matrix.command != 'make vendor' && (matrix.component == 'agent' || matrix.component == 'runk' || matrix.component == 'genpolicy') }}
```

Comment:
Could you explain why we need this? Since you're modifying a workflow YAML, GHA won't pick up this specific and will run with the unmodified workflow. You would have to create a separate PR just for this change and merge it first before this PR could use protobuf-compiler for genpolicy.

Reply:
Please ignore my previous comment. That is only valid for workflows triggered by `pull_request_target` (e.g. the k8s tests). For workflows triggered by `pull_request` like this one, GHA will run the base branch workflow YAMLs (albeit with no secrets).

Reply:
Yep. Just for visibility, this change was required to pass `[build-checks / check (genpolicy, make check)]` due to a dependency introduced in this PR https://github.com/kata-containers/kata-containers/actions/runs/8103694150/job/22149534092?pr=9185#step:13:423